var_make_debian.sh

```diff
@@ -986,6 +1023,21 @@ case $PARAM_CMD in
 	freertosvariscite )
 		cmd_make_freertos_variscite
 		;;
+	debianbase )
+		cmd_make_uboot  &&
+		cmd_make_kernel &&
+		cmd_make_kmodules &&
+		cmd_make_kernel_header_deb &&
+		cmd_make_debian_base_rootfs
+		;;
+	variscite )
+		cmd_make_uboot  &&
```

Comment:
This will not reuse the old kernel and similiar, but instead generate everything anew

Reply:
Then, could I add the condition that if the "debian_base" package already exists, it should only unpack and install the Variscite packages, and in the case of wanting to create a complete Meticulous image, a condition that checks if the packages from the previous stages already exist and then only unpack and install the Meticulous packages?